api/routes.ts

```diff
@@ -25,10 +27,14 @@ export async function setup(app:exp.Express) {
     //----------------------------------------------------------
     // Create Services
     //----------------------------------------------------------
-    let idsvc: id.IdentityService = new id.IdentityService();
+    const idDb = new nedb();
+    const idStore = new sm.Store(idDb);
```

Comment:
moving the dependency construction up the stack to the routing layer keeps the infrastructure an app level concern, insulating business logic from infrastructure choices. as long as the store returns a promise that resolves to an array of t, the persistence implementation details can be ignored.